### ISSUE TITLE
Note Interested in Recommended Tweak

### ISSUE THREAD

**Issue by J-SoG-YK** [yuliskov/SmartTubeNext#818]
My muscle memory has me selecting the Not Interested instead of Add to Playlist option of the long press context menu on the Recommended row.  Regardless, it's still easy enough to accidentally select Not Interested using a remote.  If possible, I'd like to see the Not Interested option of  either made a toggle (Not Interested/Interested) or moved below the Add/Remove to playlist and Add/Remove to playback queue options.  In fact, I seem to recall the Not Interested option being lower down the context menu in previous versions (though I could be wrong).

This could be of more concern to me, if selecting Not Interested also affects what videos are recommended, as well.  If all Not Interested does is remove a specific video from the Recommended row, this could be considered a very low priority request.

**Comment by onolox** [comment]
This is not needed, YT don't take the "not interested" seriously, I receive the content that I marked a few days latter...

It's very hard to accidentaly hit not interested dude...

**Comment by kj4z** [comment]
Actually, it's quite easy to hit the "not interested" button accidentally. I just did it myself twice in a row before I realized what was happening. I do it so quickly that I never even see the menu on screen. Muscle memory + remote.

**Comment by J-SoG-YK** [comment]
Exactly!  With the frequent unplayable video issues recently, I'd gotten really accustomed to long pressing Select, and then Select twice to quickly add videos to my Watch Later playlist.  So. anytime I want to add a video to Watch Later, that's what the muscle memory does.

I've changed the title to better reflect the reported issue.  It is a feature request after all.  Plus, I had either not realized or forgotten that Not Interested was an option throughout the SmartTubeNext Home screen.
